includes/actions/update-next-payment.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace AutomateWoo_Subscriptions;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
```

Comment:
This can be shortened:

```suggestion
defined( 'ABSPATH' ) || exit;
```